src/Components/ProjectCover/style.module.scss

```diff
@@ -22,3 +22,21 @@
     margin: 0 auto;
   }
 }
+
+.noMobileFadeContainer {
+  @include small {
+    height: auto;
+    display: block;
+    position: inherit;
+    margin-bottom: -85vh;
+    margin-top: mt();
+  }
+}
+
+.portraitOnly {
+  display: none;
+  @include small {
```

Comment:
there is a `portrait` mixin that might be a better fit for this use case.
Also `small` isn't actually a breakpoint in this toolkit because that would mean anything greater than 0px. Instead there's `small-only` which applies to < 375px. One of the super annoying quirks of scss is that it doesn't throw errors for missing mixins.

Reply:
ah, I added small max-width: 767, but i'll remove it and use portrait!